Source/Scene/Cesium3DTile.js

```diff
@@ -61,30 +67,69 @@ define([
         var contentHeader = header.content;
 
 // TODO: For the 3D Tiles spec, we want to change this to basically (x, y, z) to (x, y, z)
```

Comment:
I think we can remove this `TODO`.  The current order is more intuitive and consistent with Cesium's `Rectangle`.